README.md

```diff
@@ -21,8 +21,8 @@ Quick Start
 
 To build the JARs, use the commands:
 
-    cd lucene
-    ant -Dmobilejars=../build/<destination> clean mobile-build-modules-without-test
+    $> cd lucene
```

Comment:
Is there a reason to add `$>` to the bash code sample here? I assume people who read the README are fairly familiar with the commands here. If the intention is to highlight bash keywords, how about use the GitHub markdown (three backticks + `bash` as mentioned [here](https://help.github.com/articles/creating-and-highlighting-code-blocks/))? This comment applies to all other `$>` additions in the PR.